docs/reference/syntax.md

```diff
@@ -132,7 +132,7 @@ a table listing all syntax that is available in code mode:
 | Include module           | `{include "bar.typ"}`         | [Scripting]($scripting/#modules)   |
 | Import module            | `{import "bar.typ"}`          | [Scripting]($scripting/#modules)   |
 | Import items from module | `{import "bar.typ": a, b, c}` | [Scripting]($scripting/#modules)   |
-| Comment                  | `[/* block */, // line]`      | [Below](#comments)                 |
+| Comment                  | `{/* block */}`, `{// line}`  | [Below](#comments)                 |
```

Comment:
I'm not sure why this used to use square brackets instead of curly brackets, so there may be a problem with using curly brackets here.